<ui version="4.0" >
 <class>ChatWindow</class>
 <widget class="QWidget" name="ChatWindow" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>559</width>
    <height>445</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3" >
   <item>
    <widget class="QSplitter" name="splitter_2" >
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <widget class="QSplitter" name="splitter" >
      <property name="orientation" >
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QTextEdit" name="textEdit" >
       <property name="undoRedoEnabled" >
        <bool>false</bool>
       </property>
       <property name="textInteractionFlags" >
        <set>Qt::TextSelectableByKeyboard|Qt::TextSelectableByMouse</set>
       </property>
      </widget>
      <widget class="QWidget" name="layoutWidget" >
       <layout class="QVBoxLayout" name="verticalLayout" >
        <item>
         <widget class="QLabel" name="label" >
          <property name="text" >
           <string>Something here...</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer" >
          <property name="orientation" >
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0" >
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLabel" name="label_2" >
          <property name="text" >
           <string>Contact image</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
     <widget class="QWidget" name="layoutWidget" >
      <layout class="QVBoxLayout" name="verticalLayout_2" >
       <item>
        <widget class="QToolBar" name="toolBar" >
         <property name="minimumSize" >
          <size>
           <width>0</width>
           <height>32</height>
          </size>
         </property>
         <property name="maximumSize" >
          <size>
           <width>16777215</width>
           <height>32</height>
          </size>
         </property>
         <property name="sizeIncrement" >
          <size>
           <width>0</width>
           <height>32</height>
          </size>
         </property>
         <property name="baseSize" >
          <size>
           <width>0</width>
           <height>32</height>
          </size>
         </property>
         <property name="acceptDrops" >
          <bool>true</bool>
         </property>
         <property name="windowTitle" >
          <string>Quick Actions</string>
         </property>
         <property name="movable" >
          <bool>false</bool>
         </property>
         <addaction name="actionInsert_Emoticon" />
         <addaction name="actionNudge" />
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout" >
         <item>
          <widget class="QTextEdit" name="inputWidget" >
           <property name="enabled" >
            <bool>true</bool>
           </property>
           <property name="minimumSize" >
            <size>
             <width>0</width>
             <height>0</height>
            </size>
           </property>
           <property name="baseSize" >
            <size>
             <width>0</width>
             <height>40</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="sendButton" >
           <property name="text" >
            <string>Send Message</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
  <action name="actionInsert_Emoticon" >
   <property name="text" >
    <string>Insert Emoticon</string>
   </property>
  </action>
  <action name="actionNudge" >
   <property name="text" >
    <string>Nudge</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
